packages/calcite-components/src/components/input-time-picker/input-time-picker.tsx

```diff
@@ -819,11 +860,10 @@ export class InputTimePicker
             onCalciteInputInput={this.calciteInternalInputInputHandler}
             onCalciteInternalInputFocus={this.calciteInternalInputFocusHandler}
             readOnly={readOnly}
+            ref={this.setInputAndTransitionEl}
```

Comment:
Can you restore the comment and order of this property? We need it at the end to ensure all props are in sync by the time the ref is called.